### PR TITLE
Replaces apache thrift module source

### PR DESCRIPTION
**Pull request by beremtl**
Fixes #0000

Changes proposed in this pull request:

* Change 1
* Change 2

Output from acceptance testing:

```
$ make testacc TEST=./ovirt TESTARGS='-run=TestAccOvirtDataCenter_'

...
```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -30,6 +30,7 @@ github.com/aliyun/aliyun-tablestore-go-sdk v4.1.2+incompatible/go.mod h1:LDQHRZy
 github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239/go.mod h1:2FmKhYUyUczH0OGQWaF5ceTx0UBShxjsH6f8oGKYe2c=
 github.com/antchfx/xpath v0.0.0-20190129040759-c8489ed3251e/go.mod h1:Yee4kTMuNiPYJ7nSNorELQMr1J33uOpXDMByNYhvtNk=
 github.com/antchfx/xquery v0.0.0-20180515051857-ad5b8c7a47b0/go.mod h1:LzD22aAzDP8/dyiCKFp31He4m2GPjl0AFyzDtZzUu9M=
+github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999/go.mod h1:cp2SuWMxlEZw2r+iP2GNCdIi4C1qmUzdZFSVb+bacwQ=
```
Comment (imjoey) [inline go.sum]: Hi, in my case, after running the go mod tidy, the 10th line git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg= has been removed. Could you please recheck this? Thank you. 😄

**Comment by beremtl** [comment]
Hi, I checked again through the 'go mod tidy', and it doesn't del the 10th line in go.sum

**Comment by imjoey** [comment]
Hi, my go version is `go1.12.9 darwin/amd64`. The replaced module `git.apache.org/thrift.git` is removed after `go mod tidy` command. May I know the go version in your environment? 

BTW, is there any problems that you encountered before adding replacement? Thanks.

**Comment by beremtl** [comment]
My go version is `go1.11.8 linux/amd64`
Before I added replacing string, I have an error with `git.apache.org/thrift.git`

**Comment by imjoey** [comment]
Hi @beremtl , for consistent behavior, I would prefer to recommend you to upgrade to go1.12+. I would also update the relevant sections in readme. Could you please re-commit the `go.sum` by go1.12+? Totally sorry for the inconvenience.

Just FYI, I also notice that the thrift issue was discussed at https://github.com/hashicorp/terraform/issues/22664. There is another solution that using `GOPROXY=https://proxy.golang.org` when running go cmd/tool.

**Comment by beremtl** [comment]
Hi @imjoey  after upgrade to go1.13 everything is looks great. And error is not longer exist. Thank's, and I close this pull request

**Comment by imjoey** [comment]
Hi @beremtl , as far as I can tell, go1.13 works fine because `GOPROXY=https://proxy.golang.org` is enabled by default since go1.13. It's a module mirror for accelerating Go module downloads, an index for discovering new modules. While the missing `git.apache.org/thrift.git` has been cached on it.

There are two upstream patches related to this issue, https://github.com/hashicorp/go-getter/pull/204 and https://github.com/hashicorp/terraform/pull/22741, have been merged, which removes the dependency of `git.apache.org/thrift.git`. So I think that would be the final perfect solution.

But now we still need to support go1.12 users. So would you please reopen this pr and update the `go.sum` by removing the replaced module `git.apache.org/thrift.git`, just as mentioned above? Then we could merge this pr. Thank you very much.

**Comment by imjoey** [comment]
@beremtl  Many thanks.

**Pull request merged**